packages/commercetools/composables/src/useCart/actions.ts

```diff
@@ -0,0 +1,24 @@
+import { ProductVariant } from "@vue-storefront/commercetools-api/lib/src/types/GraphQL"
+import { UiCartProduct } from '@vue-storefront/interfaces'
+
```

Comment:
please add that 'action| is a name from ct ideally with link, for people new to this platform that will want to maintain this file naming could be tricky